invoice/views.py

```diff
@@ -193,6 +195,21 @@ class InvoiceCreateView(LoginRequiredMixin, SuccessMessageMixin, CreateView):
     def get_success_url(self):
         return reverse("invoice-update", kwargs={"pk": self.object.id})
 
+    def form_valid(self, form):
+        invoice_number = form.cleaned_data.get("invoice_number")
```

Comment:
Do we want to allow the user to manually set the invoice number?
If yes this is fine, if not we should remove the field from the UI

Reply:
I would prefer no human input for the invoice numbers.

Reply:
removed it